### PR TITLE
feat(PluggyClient): add credentials encryption for POST/PATCH to /items and add rsaPublicKey method. 

### PR DESCRIPTION
**Pull request by NicolasMontone**

### CODE REVIEW

File: src/main/java/ai/pluggy/client/auth/EncryptedParametersInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            throw new IOException("Error encrypting parameters", error);
+        }
+    }
+
+    private JsonObject transformBodyToJsonObject(RequestBody body) {
+        Buffer buffer = new Buffer();
+        try {
+            body.writeTo(buffer);
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (ghostframe) [inline src/main/java/ai/pluggy/client/auth/EncryptedParametersInterceptor.java:101]: Change for a rethrow since this is not a handled error

**Comment by ghostframe** [comment]
EXELENTE

**Review by ghostframe — APPROVED** [review]

File: src/main/java/ai/pluggy/client/auth/EncryptedParametersInterceptor.java (hunk 1 above)
Comment (NicolasMontone) [reply src/main/java/ai/pluggy/client/auth/EncryptedParametersInterceptor.java]: You are right, thanks!

**Review by tmilar — COMMENTED** [review]
✅

**Pull request merged**